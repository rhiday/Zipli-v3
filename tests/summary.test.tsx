// ðŸš¨ Snapshot test to freeze the donation summary UI layout. If this test fails, review any UI changes carefully before updating the snapshot.
import React from 'react';
import { render } from '@testing-library/react';
import SummaryPage from '../src/app/donate/summary/page';

describe('SummaryPage UI', () => {
  it('renders the summary UI and matches the snapshot', () => {
    const { container } = render(<SummaryPage />);
    expect(container).toMatchSnapshot();
  });
}); 