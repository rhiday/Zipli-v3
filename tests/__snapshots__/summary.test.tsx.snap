// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SummaryPage UI renders the summary UI and matches the snapshot 1`] = `
<div>
  <div
    class="flex flex-col min-h-screen bg-white max-w-md mx-auto items-center justify-center gap-4"
  >
    <p
      class="text-gray-600"
    >
      No donation items found.
    </p>
    <button
      class="inline-flex items-center justify-center rounded-full font-semibold transition-colors focus:outline-none focus-visible:outline-2 focus-visible:outline-offset-2 focus-visible:outline-interactive cursor-pointer gap-2.5 shrink-0 disabled:bg-inactive disabled:text-tertiary disabled:cursor-not-allowed bg-positive hover:bg-positive/90 px-6 py-3 text-bodyLg"
    >
      Start New Donation
    </button>
  </div>
</div>
`;
